tinymce/tinymce
# Update jscripts/tiny_mce/tiny_mce_popup.js

The timeout for resizeToInnerSize is not enough for Google Chrome to resize the popup corretly.
This will cause any popup appear or maximized or minimized.
I changed the timeout to 50.

This is how to reproduce:
1. Open a page with WYSIWYG
2. Click Insert/Edit image tool
3. Close the tool.
4. Open the the same tool or other tool that invokes a popup - you should get a wrong size popup

You have to use Chrome and inline popups deactivated to reproduce this.
